geneticalg/examples/run_ga/run_ga.py

```diff
@@ -1,23 +1,39 @@
 import pandas as pd
-from geneticalg import gen_chromosome, calc_fitness_all, mutate
+from geneticalg import (gen_chromosome,
+                        calc_fitness_all,
+                        mutate,
+                        crossover,
+                        list_duplicates,
+                        del_duplicates,
+                        sort_chromosomes)
 
 video_data = pd.read_csv("./data/video_ts_data.csv", index_col=None)
 video_lengths = pd.read_csv("./data/video_lengths.csv").iloc[:, 0].values
-video_list = pd.read_csv("./data/video_list.csv",
-                         index_col=None).iloc[:, 0].values
+video_list = pd.read_csv("./data/video_list.csv", index_col=None).iloc[:, 0].values
```

Comment:
FWIW, still see no `data/` in this branch either. It got removed in 77288ec2b5d3cdeb2933c27ad2f07f4fd62b4336 I believe